tpm2/tpm2.go

```diff
@@ -445,15 +466,38 @@ func decodeCreateKey(in []byte) ([]byte, []byte, error) {
 // CreateKey creates a new key pair under the owner handle.
 // Returns private key and public key blobs.
 func CreateKey(rw io.ReadWriter, owner tpmutil.Handle, sel PCRSelection, parentPassword, ownerPassword string, pub Public) ([]byte, []byte, error) {
-	cmd, err := encodeCreate(owner, sel, parentPassword, ownerPassword, pub)
+	cmd, err := encodeCreate(owner, sel, parentPassword, ownerPassword, []byte{} /*inSensitive*/, pub)
 	if err != nil {
 		return nil, nil, err
 	}
 	resp, err := runCommand(rw, tagSessions, cmdCreate, tpmutil.RawBytes(cmd))
 	if err != nil {
 		return nil, nil, err
 	}
-	return decodeCreateKey(resp)
+	return decodeCreate(resp)
+}
+
+// Create a data blob object that seals the sensitive data under a parent and with a
+// password and auth policy. Access to the parent must be available with a simple password.
+// Returns private and public portions of the created object.
+func Seal(rw io.ReadWriter, parentHandle tpmutil.Handle, parentPassword, objectPassword string, objectAuthPolicy []byte, sensitiveData []byte) ([]byte, []byte, error) {
```

Comment:
How about new `func Create` that has the common code from `CreateKey` and `Seal`. And make these two just thin wrappers around it.

Reply:
done